pkg/build/api/types.go

```diff
@@ -707,6 +739,12 @@ type BuildRequest struct {
 
 	// Env contains additional environment variables you want to pass into a builder container
 	Env []kapi.EnvVar
+
+	// SkipBuildsInQueue will cause all builds in the queue marked as skipped and
+	// make the build created from this build requests be the only build in the
+	// queue. This option is valid only for build configurations with "recent"
+	// build scheduling policy.
+	SkipBuildsInQueue *bool
```

Comment:
What does Jenkins call this flag?  How many flags do they have?  What about jenkins pipeline and concourse?